### ISSUE TITLE
Payer bank verified status

### ISSUE THREAD

**Issue by VitoRomaniello** [paypal/PayPal-NET-SDK#281]
### General information
* SDK/Library version:1.8.0
* Environment: Sandbox
* Language, language version, and OS: ASP.NET MVC 5, C#

### Issue description

Scope: avoid that unverified bank payers pay on our site

Scenario:
Created an account on the sandbox and setup his bank verified status to false as in the following screenshot

![immagine](https://user-images.githubusercontent.com/19202598/29524724-e6a40f42-8690-11e7-9dba-a0384470b9ca.png)

When making the payment I would like to filter those accounts that are bank unverified. I am then trying to call 

```
var paymentDetails = Payment.Get( apiContext, paymentID );
if ( paymentDetails != null ) {
    if ( paymentDetails.payer.status == PAYER_VERIFIED ) {
    }
}

```

However the returned payer status is always VERIFIED. How can I do to achieve the explained scenario?

Update: i have noted that when I add a credit card to the account it always change its status to verified.

**Comment by xiaoleih41** [comment]
You won't be able to get this type of account information from another account. Please consider integrating [log in with PayPal](https://developer.paypal.com/docs/integration/direct/identity/log-in-with-paypal/) and set the proper [user attributes](https://developer.paypal.com/docs/integration/direct/identity/attributes/) to check the verified status of a PayPal account.

**Comment by VitoRomaniello** [comment]
Hello @xiaoleih41 
thanks for your answer. 

I am not really sure that I completely got your answer correctly. I can not force my customers to log in with paypal even because not everybody has a Paypal account. 

Let me rephrase better what I have done until now. Have a look at the following scenarios:

**User without an associated credit card and Bank verified account = No**
In this case I correctly get "UNVERIFIED" before confirming the payment.

**User with an associated credit card and Bank verified account = No**
In this case I always get "VERIFIED" despite the fact that I set Bank Verified Account = NO. As previously written in fact, If I add a credit card for the user, the sandbox UI always set the Bank Verified Account status to YES and I hve no way to modify the value.

The reason behind this request is tied to the fact that my company sells digital goods and services which are not covered by the merchant protection. In the past months we've got some chargebacks from users that:
1. create a new paypal account and add a (fake, duplicated, stolen) credit card as a funding instrument
2. buy goodies on our digital store
3. The real owner of the credit card open a chargeback on its bank as soon as he knows that he does not recognize a transaction.

This is mainly due to the fact that PayPal does not force the credit card ownership verification before than 25/30 days or after spending more than X. For this reason we would like to address this behaviour by avoiding users that are not still verified to use those funding instruments. 

could you please suggest how to achieve our scope?

**Comment by xiaoleih41** [comment]
the verified status is not specifically tired to a card verification or a bank verification. It's for a verified PayPal account. From API standpoint, there's no better way to tell if a stolen card is added to a stolen verified PayPal account. PayPal customer service may have more fraud prevention suggestions for digital goods transaction.

**Comment by bluk** [comment]
Closing due to inactivity. If this is still an issue, please contact [PayPal Support](https://developer.paypal.com/support/) who can help as noted in the last comment. Thanks!
